frontend/src/utils/createShoot.js

```diff
@@ -335,3 +343,18 @@ export function getWorkerProviderConfig (infrastructureKind) {
     }
   }
 }
+
+export function alignAndReturnNodeCIDR (shootResource) {
+  const networks = get(shootResource, 'spec.provider.infrastructureConfig.networks')
+  const networkCIDR = get(networks, 'vpc.cidr', get(networks, 'vnet.cidr', get(networks, 'workers')))
+  const nodeCIDR = get(shootResource, 'spec.networking.nodes')
+  if (networkCIDR && networkCIDR !== nodeCIDR) {
+    // networkCIDR is the leading CIDR as it is more likely to be modified by the user
+    // nodeCIDR needs to be a subset or equal to networkCIDR. Usually they should be equal
```

Comment:
> nodeCIDR needs to be a subset or equal to networkCIDR

what if nodeCIDR is a subset of the networkCIDR, in this case you would just reset it to the networkCIDR